services/core-api/app/api/projects/information_requirements_table/resources/Information_requirements_table_import.py

```diff
@@ -0,0 +1,31 @@
+from flask_restplus import Resource
+from flask import request, current_app
+from werkzeug.datastructures import FileStorage
+from app.extensions import api
+from app.api.utils.resources_mixins import UserMixin
+from app.api.utils.custom_reqparser import CustomReqparser
+from app.api.projects.project.models.project import Project
+from app.api.mines.mine.models.mine import Mine
+from app.api.services.document_manager_service import DocumentManagerService
+
+
+class InformationRequirementsTableListImportResource(Resource, UserMixin):
```

Comment:
This looks to be the resource that relates to the document upload for the spreadsheet. We may want to rename the class to reflect that and also update the API doc description to be a bit clearer to the developers what this is doing.  Unless this is doing something else that is not obvious?

Additionally we may want to perform checks on both project and mine to make sure they exist before using the DocumentManagerService.

Lets also add "params" to the @api.doc object to be consistent with our other endpoints.